mantidproject/mantid
# Change output name in fit browser for multiple plots of the same workspace

If more than one plot of the same workspace is open this PR changes the default output name in the fit property browser so the resulting output workspaces are not overridden  

**To test:**
Open workbench and load some data
Generate two plots from the same spectra (data)
Do a fit on the first one
Do a fit on the second one
There should be two sets of fitting workspaces.

Fixes #27288 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
